dm/config/subtask.go

```diff
@@ -429,3 +429,11 @@ func (c *SubTaskConfig) Clone() (*SubTaskConfig, error) {
 
 	return clone, nil
 }
+
+// AdjustOnlineDDL adjust OnlineDDL through OnlineDDLScheme.
+func (c *SubTaskConfig) AdjustOnlineDDL() {
+	if c.OnlineDDLScheme == GHOST || c.OnlineDDLScheme == PT {
+		c.OnlineDDL = true
+	}
+	c.OnlineDDLScheme = ""
```

Comment:
the original value should not be changed, to support downgrading

Reply:
+1

Reply:
It does not affect the original configure...